models/meteora/meteora_trades.sql

```diff
@@ -0,0 +1,159 @@
+ {{
+  config(
+        schema = 'meteora',
```

Comment:
should we make:
`meteora_solana` to be more clear?

if so, also update file name, schema files, downstream refs, etc.